data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,734 @@
+
+mission "You Are A Pirate"
+	landing
+	source
+		government "Pirate"
+	to offer
+		not "chosen sides"
+		or	
+			and
+				or
+					"reputation: Republic" < -10
+					"reputation: Syndicate" < -10
+					"reputation: Free Worlds" < -10
+				"reputation: Merchant" < -10
+			"reputation: Pirate" > 20
+	on offer
+		conversation "You Are A Pirate"
+	on accept
+		event "became a pirate"
+		fail
+
+conversation "You Are A Pirate"
+	`Preparing to land is second nature by now: patch any perforations in the inner hull, get in touch with ground control, and then make sure your chosen parking pad isn't controlled by extortionists. When you disembark it's with a pistol at your hip, bribe money in your pocket, and a swagger in your step; to leave behind any one of these is to invite trouble. Perhaps a life of violence on the high skies and association with the rougher elements of the galaxy weren't what you had in mind when you left your homeworld, but it's what you found.`
+	choice
+		`	(This is my life now. Piracy, smuggling, the existence of the outlaw. I'm ready to embrace it.)`
+		`	(This isn't what I wanted. This isn't the life I imagined when I decided to take to the high skies.)`
+			decline
+	`	Resolve will carry you far in this world. Tonight it's carried you to a spaceport paint supplier; you return to your vessel with a pallet loaded with blacks and reds. Apply it to your vessel, make your intentions known to the galaxy. There can be no pretense of peace when you fly girded in blood and midnight, but maybe that's exactly the way you want it to be.`
```

Comment:
If the player is reading this they've already made up their mind, no need to render it in subjunctive - and it would ruin the flow.

I do like the credit cost idea, though.

Reply:
I don't think it ruins the flow myself - the following sentence includes a hypothetical anyways. But up to you!